update geometry/polar-coordinate (#99)
OI-wiki/i18n: docs/geometry/polar-coordinate.md

author: Ir1d, HeRaNO, Chrogeek, abc1763613206

## Polar coordinates and polar coordinate system

> > This paragraph is taken from chapter 4-4 high school mathematics electives, people's education edition.

Reviewer: This is the content of the book *Senior High School Math: Elective 4-4* by *People's Education Press*

Let's consider the actual scenario, such as sailing. We say "$B$ is in the direction of $30^\circ$ north east of $A$ , and the distance is $100$ meters" instead of "using $A$ as the origin to create a rectangular coordinate system, $B(50,50\sqrt 3)$ ".

In this way, we choose a certain point $O$ on the plane, called **pole**, and draw a ray $Ox$ from the pole, called **polar axis**. Then choose a unit length (usually in mathematical problems it is $1$ ), an angle unit (usually radians), and its positive direction (usually counterclockwise), thus a **polar coordinate system** is established.

How do we describe the position in the polar coordinate system?

> Let $A$ be a point on the plane, and the distance $|OA|$ , denoted as $\rho$, between the pole $O$ and $A$ is the **polar diameter**, taking the $\angle xOA$ , denoted as $\theta$ , starting from the polar axis, and ending at the edge $ OA$ as **polar angle**, then the ordinal pair $(\rho,\theta)$ is the **polar coordinate of $A$**.

Reviewer: ```suggestion
Let $A$ be a point on the plane, and the distance $|OA|$ , denoted as $\rho$, between the pole $O$ and point $A$ is the **polar diameter**. Take the $\angle xOA$ , denoted as $\theta$ , which starts from the polar axis and ends at the edge $OA$, as **polar angle**, then the ordinal pair $(\rho,\theta)$ is the **polar coordinate of $A$**.
```

According to the definition of the same angle at the end, the point represented by $(\rho,\theta)$ and $(\rho,\theta+2k\pi)\ (k\in \mathbb{Z})$ is actually the same. In particular, the polar coordinate of the pole is $(0,\theta)\ (\theta\in \mathbb{R})$, so there are countless kinds of polar coordinates of points in the plane.

> If $\rho>0,0\le \theta<2\pi$ is specified, then in addition to the poles, points in other planes can be represented by a unique ordinal pair $(\rho,\theta)$ , and points represented by polar coordinate $(\rho,\theta)$ is uniquely determined.

Reviewer: ```suggestion
If $\rho>0,0\le \theta<2\pi$ is specified, then in addition to the poles, points in other planes can be represented by a unique ordinal pair $(\rho,\theta)$ , and points represented by polar coordinate $(\rho,\theta)$ is unique.
```

> Of course, sometimes it is not very convenient to study the graphics in the polar coordinate system. If we want to study it in the rectangular coordinate system, we can use the mutualization formula.

Author: ```suggestion
Of course, sometimes it is not very convenient to study the graphics in the polar coordinate system. If we want to study it in the rectangular coordinate system, we can use the mutualization equation.
```

Reviewer: emmmm 
[conversion formula](https://en.wikipedia.org/wiki/Polar_coordinate_system#Converting_between_polar_and_Cartesian_coordinates) 可能更好？

> Of course, sometimes it is not very convenient to study the graphics in the polar coordinate system. If we want to study it in the rectangular coordinate system, we can use the mutualization formula.

Reviewer: 互化给个 ref？

Author: > 互化给个 ref？

mutualization formula是在网上没找到直接翻的 💀 不过想了下 mutualization equation会更好？
